.github/actions/latest-stable-tag/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "latest-stable-tag",
+  "version": "1.0.0",
+  "description": "Gets the latest stable tag in a Git repository",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules npx jest"
+  },
+  "author": "",
```

Comment:
Take credit, man.

Reply:
Well, I have no intention of releasing this for public use, so I left it blank (I just did `npm init` and pressed Enter for each prompt...)

Reply:
I removed this property, since it's not strictly necessary